setup/install_gatling/30_classification/50_classify_nodes_via_site_pp.rb

```diff
@@ -1,4 +1,23 @@
-test_name "Classify Puppet agents on master"
-skip_test 'Installing PE, not FOSS' unless ENV['BEAKER_INSTALL_TYPE'] == 'foss'
+test_name "Classify Puppet agents on master" do
+  skip_test "Only runs on FOSS" unless ENV["BEAKER_INSTALL_TYPE"] == "foss"
 
-classify_foss_nodes(master)
+  # Ensure that classifier data sources are removed hiera configs on foss
```

Comment:
"Ensure that classifier data sources are removed (from the) hiera configs on foss"?

Reply:
Grammar corrected. :+1: